src/Microsoft.Extensions.FileProviders.Physical/Internal/FileSystemInfoHelper.cs

```diff
@@ -3,15 +3,11 @@
 
 using System.IO;
 
-namespace Microsoft.Extensions.FileProviders
+namespace Microsoft.Extensions.FileProviders.Physical.Internal
```

Comment:
Should be fine to leave it in the root namespace since the type's internal. Alternatively move it to the `.Internal` namespace and leave the type public (I prefer the former).